src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -106,8 +106,8 @@ public void CopyTo_MultidimensionalArray_ThrowsIndexOutOfRangeException()
             AssertExtensions.Throws<ArgumentException>(null, () => listener.Prefixes.CopyTo(new object[1, 1], 0));
         }
 
-        [ActiveIssue(22055, TargetFrameworkMonikers.UapAot)]
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Arrays with non-zero lower bounds are not supported for NetNative")]
```

Comment:
Why is this a SkipOnTargetFramework?  Did you discuss this with .NET Native experts?  I'm just wondering if this should remain as an issue and have the .NET Native team investigate the bug.